client_utils/include/hawktracer/client_utils/tcp_client_stream.hpp

```diff
@@ -17,7 +17,7 @@ namespace ClientUtils
 class TCPClientStream : public parser::Stream
 {
 public:
-    TCPClientStream(const std::string& ip_address, uint16_t port, bool wait_for_server = true);
```

Comment:
I don't think this is necessary. It is affecting other parts of this repo.

Reply:
Okay, I'm super confused here – we either change the default to false here OR somehow allow this to be passed in. 

If wait_for_server is true, this thing will poll indefinitely

Reply:
Removed